examples/inference/lisa_smbhb/lisa_smbhb_relbin.ini

```diff
@@ -39,7 +39,7 @@ spin1z = 0.597755394865021
 spin2z = 0.36905807298613247
 distance = 17758.367941273442
 inclination = 1.5970175301911231
-t_obs_start = -31536000
+t_obs_start = 31536000
```

Comment:
I think the argument here was that a negative `t_obs_start` doesn't make physical sense. So why did the code work when it was negative (or at least give sane-looking likelihoods). What was happening in the previous case that is different now?

Reply:
@spxiwh Thanks for these suggestions! This PR is separated from my PR in the BBHx plugin, so this will not slow down our LDC submission. For this `t_obs_start` issue, you can see BBHx will check if it's positive https://github.com/mikekatz04/BBHx/blob/master/bbhx/waveformbuild.py#L441, if it's negative, `t_start` will be set to `np.zeros(self.num_bin_all)`.

Reply:
Thanks @WuShichao ! What does it mean to the waveform that is produced if it is set to this `np.zeros` value? What is the physical start time in this case??

Reply:
@spxiwh I have searched the entire code of BBHx, it passes `t_start` to `from pyWaveformBuild_cpu import direct_sum_wrap`, but unfortunately, it is not in a readable format, I believe the magic happens in that function.